jenkins/scripts/artifactory/utils.sh

```diff
@@ -21,7 +21,8 @@ rt_upload_artifact() {
   ANONYMOUS="${3:-1}"
 
   _CMD="curl \
-    $( ([[ "${ANONYMOUS}" != 1 ]] && echo " -u${RT_USER:?}:${RT_TOKEN:?}") || true) \
+    --fail-with-body \
+    $( ([[ ""${ANONYMOUS}" -ne 1 ]] && echo " -u${RT_USER:?}:${RT_TOKEN:?}") || true) \
```

Comment:
I also now realized how bad this line of code actually is, but let's fix that in another PR.